solo/methods/base.py

```diff
@@ -506,21 +512,34 @@ def validation_step(self, batch: List[torch.Tensor], batch_idx: int) -> Dict[str
                 the classification loss and accuracies
         """
 
-        X, targets = batch
-        batch_size = targets.size(0)
+        if dataloader_idx == self.val_loader_keys.index("augmented"):
```

Comment:
I think we can find the indexes a priori with something like `on_train_start` no?